jib-core/src/main/java/com/google/cloud/tools/jib/api/Containerizer.java

```diff
@@ -235,6 +237,18 @@ public Containerizer setAllowInsecureRegistries(boolean allowInsecureRegistries)
     return this;
   }
 
+  /**
+   * Sets whether or not to run the build in offline mode. In offline mode, the base image is
+   * retrieved from the cache instead of pulled from a registry.
+   *
+   * @param offline if {@code true}, the build will run in offline mode
+   * @return this
+   */
+  public Containerizer setOfflineMode(boolean offline) {
```

Comment:
The Javadoc says "In offline mode, the base image is retrieved from the cache instead of pulled from a registry". If we were going to fail `jib:build` with the offline mode, we should prominently document that too.